src/uikit.cs

```diff
@@ -17413,7 +17511,9 @@ interface UIPrintInteractionControllerDelegate {
 		UIPrinterCutterBehavior ChooseCutterBehavior (UIPrintInteractionController printInteractionController, NSNumber [] availableBehaviors);
 	}
 
-	[NoTV]
```

Comment:
Was this intentional?

Reply:
The diff is tricking you :) It was not removed, the delegate was added and then NoWatch was added to the attr. It is in the line 17516 now